lib/scry/scraper.rb

```diff
@@ -1,5 +1,42 @@
+require "rubygems"
+require "mechanize"
+require "scry/sidekiq/workers/export_generator"
+require "scry/helpers"
+require "scry/course"
+
 module Scry
-  def self.scrape(output_directory)
-    output_directory
+  extend Scry::Helpers
+
+  ##
+  # Creates sidekiq jobs for each course to generate an export.
+  #
+  # Logs in the user and goes over every course
+  # and creates a sidekiq to generate an export for it.
+  ##
+  def self.scrape(url, login, passwd, dir)
+    agent = Mechanize.new do |secret_agent_man|
+      secret_agent_man.follow_meta_refresh = true
+    end
+
+    agent.get(url) do |home_page|
+      index_page = home_page.form_with(name: "login") do |form|
+        form.fields[0].value = login
+        form.fields[1].value = passwd
```

Comment:
As mentioned before, a constant for the `0` and `1` would avoid magic numbers and help clear up the intention here.